scipy/stats/_distn_infrastructure.py

```diff
@@ -1440,9 +1440,9 @@ class rv_continuous(rv_generic):
     location, and scale parameters returning a "frozen" continuous RV object:
 
     rv = generic(<shape(s)>, loc=0, scale=1)
-        frozen RV object with the same methods but holding the given shape,
+        stats.distributions.rv_frozen object with the same methods but holding the given shape,
```

Comment:
Better, I was thinking of:

```
`rv_frozen` object with
```

I _think_ the class name needs to go in backticks.

Reply:
I think you're right.

At line 1316 there's

>     `rv_continuous` is a base class ...

​In HTML we see

> [rv_continuous][1] is a base class ...

[1]:https://docs.scipy.org/doc/scipy/reference/generated/scipy.stats.rv_continuous.html#scipy.stats.rv_continuous